website/pages/blog/graphiql-3/index.mdx

```diff
@@ -0,0 +1,212 @@
+---
+title: GraphiQL 3 Is Here and GraphiQL 4 Is Coming
```

Comment:
I would change the title to something that also gives information about the content
maybe like "GraphiQL v3 is here - React 18, better codebase, balblabla, and you can try v4 alpha with Monaco editor!"